util/docker-container/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:22.04
+
+RUN apt-get update -y
+RUN apt-get install -y gcc gcc-multilib nasm make build-essential
+RUN apt-get install -y binutils
+RUN apt-get install -y gdb
+RUN apt-get install -y vim nano
+RUN apt-get install -y tmate
+RUN apt-get install -y python2.7
+RUN apt-get install -y python3.10 python-is-python3
+
+CMD [ "bash" ]
```

Comment:
```suggestion
RUN apt-get update \
    && apt-get install -y \
        gcc \
        gcc-multilib \
        nasm \
        make \
        build-essential \
        binutils \
        gdb \
        vim \
        nano \
        tmate \
        python2.7 \
        python3.10 \
        python-is-python3 \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*

CMD [ "bash" ]
```

Reply:
[Here] are some best practices about Dockerfile. We need to put all these commands inside a single one in order to have fewer layers.

[Here]: https://docs.docker.com/develop/develop-images/instructions/